### ISSUE TITLE
Error with esc key handling in calcite.js 

### ISSUE THREAD

**Issue by kellyhutchins** [Esri/calcite-web#992]
Looks like there is an issue in calcite-js with the key handling for the esc key.  

Reproducible steps: 
1. Navigate to the content page on ArcGIS Online. 
2. Open the dev console and press the esc key and note the following error in the dev console: 
```main.js:3341 Uncaught TypeError: Cannot read property 'getAttribute' of null
    at main.js:3341
    at Array.forEach (<anonymous>)
    at g (main.js:3341)
    at b (main.js:3349)
    at F.emit (main.js:3368)
    at HTMLDocument.<anonymous> (main.js:3369)
```

**Comment by paulcpederson** [comment]
Looks like this is related to the `closeDrawer` logic. When there is no drawer, the handler is still attached, so you get an error. Targeting next minor release.

**Comment by paulcpederson** [comment]
Fixed, will be part of 1.2.0
